pkg/monitoring/fcdclient/go_generate.go

```diff
@@ -0,0 +1,3 @@
+package fcdclient
+
+//go:generate mockery --name Client --output ./mocks/
```

Comment:
can just have this above the interface def

Reply:
agreed! but I'm keeping it consistent with other parts of this package. I'll update in a separate PR.